rpc/http.go

```diff
@@ -212,17 +206,17 @@ func httpTimestamp(t time.Time) []byte {
 	t = t.UTC()
 	yy, mm, dd := t.Date()
 	hh, mn, ss := t.Clock()
-	day := days[3 * t.Weekday():]
```

Comment:
I am puzzled about the whole thing actually, why are we reimplementing the HTTP server?